### DISCUSSION TITLE
Changing Meeting Type (Global)

### DISCUSSION THREAD

**Discussion by cmaddon** [code4recovery/12-step-meeting-list#756] [Q&A] [answered]
Hello - I'd like to revise the checkbox meeting type on the 'Add Meeting' function from LGBTQ to LGBTQIA globally for existing meetings with LGBTQ selected and of course future meetings.

Thank you,
Curt

**Comment by joshreisner** [comment]
thanks @cmaddon , i’ve moved this to the spec repository, where we track changes to meeting types. 

could you tell us your reasoning? this could help us make the case to the general service office. 

would this be is in reference to AA? or to all programs?

**Comment by cmaddon** [comment]
This is simply in regards to the plugin we are using for our city's intergroup site. We'd like to revise to the term being utilized now on our site only. Is this plugin integrated with GSO and therefore these meeting types set by them? If so, then we aren't looking to have that effect.

**Comment by joshreisner** [comment, marked as answer]
ah ok! i'll transfer the issue back. sorry i misread. 

changing it on your site: do you have access to add code to your theme's functions.php file? if so, then you can change the meeting type how you want by adding this code. this will have the intended effect of changing existing and future meetings to use this label

```
if (function_exists('tsml_custom_types')) {
    tsml_custom_types(array(
        'LGBTQ' => 'LGBTQIA',
    ));
}
```

**Reply by joshreisner** [reply]
@cmaddon this code definitely works now! i've added it to our demo website: https://demo.code4recovery.org/meetings/attitude-adjustment-3

things to check:
* double-check the spelling of LGBTQ
* check that it's installed on the active theme's functions.php
* check that the code isn't inside an if statement

**Reply by joshreisner** [reply]
if you are using a caching plugin then that could affect the display of the site, try refreshing the cache

**Comment by cmaddon** [comment]
I do have access to functions.php, but when I added the code above (before posting this thread) it did not add a checkbox with LGBTQIA or modify the LGBTQ item already there.

**Reply by joshreisner** [reply]
oops there is a typo, change the first half to say `LGBTQ` instead of `LGTBQ`, that should fix it

**Comment by cmaddon** [comment]
Sorry, the code I pasted into functions.PHP was correct, what I posted to
you was not. The same problem remains.



On Tue, Apr 12, 2022 at 10:22 AM Josh Reisner ***@***.***>
wrote:

> oops there is a typo, change the first half to say LGBTQ instead of LGTBQ,
> that should fix it
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/756#discussioncomment-2552570>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYVER7NSLF3DYDE45CDNLKDVEWBITANCNFSM5THFHTJA>
> .
> You are receiving this because you authored the thread.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/756/comments/2552570@
> github.com>
>
-- 



Curt Maddon  | President | MaddMarketing, LLC
11816 Inwood Rd PMB 70335 | Dallas, TX 75244 | tel (800) 315-0150
| mobile (214) 215-1525 | fax (214) 292-9631


*Confidentiality Note:* This e-mail and any attachments are confidential
and may be protected by legal privilege. If you are not the intended
recipient, be aware that any disclosure, copying, distribution or use of
this e-mail or any attachment is prohibited. If you have received this
e-mail in error, please notify us immediately by returning it to the sender
and delete this copy from your system. Thank you for your cooperation.

**Comment by cmaddon** [comment]
Morning Josh - just a follow up that the problem has been resolved, thank you for your help!

**Comment by TSWrightSF** [comment]
I'm getting the same pushback.  

We have a LGBTQIA meeting that feels that having "gay" and "Lesbian" boxes ticked is insulting to their meeting participants that don't identify as Gay or Lesbian.

**Reply by joshreisner** [reply]
hi @TSWrightSF what's your website? are you AA?

if so, it sounds like a slightly different issue - you can always uncheck Gay and Lesbian if they are selected and apply to the overall existing LGBTQ type. does that fix satisfy your users? or what solution do they have in mind?

**Reply by TSWrightSF** [reply]
Yep, we're AA - santafeaa.org

I think they are sensitive about leaving behind the "Allies", the "Asexual" and the "Intersex".  Their group seems comfortable with just appending LGBTQIA to the meeting name.  I also added it to the notes box.

They seem happy with that resolution.

Thanks for all your great work, Josh -

Spencer

**Reply by joshreisner** [reply]
thanks Spencer for the site link. there is an available type called `LGBTQ` that it seems you're not using yet. you may need to expose it on the edit screen by clicking "show more." 

you can change `LGBTQ` to be `LGBTQIA` by following the instructions in [this comment above](https://github.com/code4recovery/12-step-meeting-list/discussions/756#discussioncomment-2547148).

also just wanted to check in on the color of the dropdown menu text at aasantafe.org/meetings - i'm seeing white-on-white, are you seeing that too? we typically having this gray-on-white ([demo](https://tsml-ui.code4recovery.org/tests/aasanjose.html)) - if that's something you wanted to fix it looks like your theme "access press" is setting all button colors to white with the `!important` flag. you should be able to correct it by adding this custom css:

```
#tsml-ui button:not(.bg-secondary):not(.active):not(:hover) {
  color: #6c757d !important;
}
```

**Comment by TSWrightSF** [comment]
Thanks, Josh -

I could not figure out how to make the TMSL UI to work without that
white-on-white issue occurring.  That css snippet works perfectly.

We should probably move to a different theme.  It is a legacy issue (with
some customization) and we don't want to break anything.

Thank you!
